### DISCUSSION TITLE
get_contributers

### DISCUSSION THREAD

**Discussion by Nuzair46** [PyGithub/PyGithub#1864] [General]
Is there a function to get contributors of a repository. I just read the docs and couldn't find anything related to this.

**Comment by ghost** [comment]


**Reply by Nuzair46** [reply]
Bruh. I know how to do that.

I was asking if there is a function that I could call in my code to get the contributors of any repo. I was not asking how to find the contributors of THIS repo.

**Reply by Nuzair46** [reply]
I haven't seen any function for that in the docs. thats why I asked here.

**Comment by kaustubhgupta** [comment]
This should help:
https://pygithub.readthedocs.io/en/latest/github_objects/Repository.html#github.Repository.Repository.contributors_url
